packages/ui/components/core/Link.tsx

```diff
@@ -0,0 +1,34 @@
+import NextLink from 'next/link'
+
+import { Text, createStyles } from '@mantine/core'
+
+const useStyles = createStyles((theme) => ({
+	link: {
+		color: theme.other.colors.secondary.black,
+		paddingBottom: theme.spacing.sm,
+		paddingTop: theme.spacing.sm,
+		paddingLeft: theme.spacing.xs,
+		paddingRight: theme.spacing.xs,
+		borderRadius: theme.spacing.sm,
+		textDecoration: 'underline',
+		'&:hover': {
+			backgroundColor: theme.other.colors.primary.lightGray,
+			textDecoration: 'none',
+		},
+	},
+	text: {},
+}))
+
+export const Link = ({ children, href }: Props) => {
+	const { classes } = useStyles()
+	return (
+		<NextLink href={href} className={classes.link}>
+			<Text className={classes.text}>{children}</Text>
+		</NextLink>
+	)
+}
+
+type Props = {
+	children: string
+	href: string
+}
```

Comment:
I removed the 'size' prop - any changes in sizing should be handled through CSS/theming - if we find that there are instances where we need different sizes, we can create different variants of the component (like the Button variants, for example)